### ISSUE TITLE
Create basic representation

### ISSUE THREAD

**Issue by MatthewJOrr** [johndavidbustard/RoughWorld#1]
An important major task would be to create a version of the project that can be viewed and understood by others. This could be merely be a basic environment where a model could walk around a room will several different objects.

There does not necessarily need to be a visual representation here, as all interactions could be shown through text.

**Comment by johndavidbustard** [comment]
I agree. I have started uploading some code to make a step in this direction. Until marking etc. is complete progress will be slow. At this stage it is valuable for people to create their thoughts on breaking down common human activities (like the example that has been started in the wiki)
